R/lancelot.R

```diff
@@ -2270,6 +2344,14 @@ lancelot_check_data <- function(data) {
                "care homes/community"))
   }
 
+  ## NOTE: I (RGF) think that nms_deaths_aggr might really be deaths
+  ## and nms_deaths_split more completely, but practically I expect
+  ## this is equivalent.
+  ##
+  ## NOTE: This is asserted at the level of the _entire_ time series,
+  ## not per day; that might be overly strict?
+  nms_deaths_ages <- paste0("deaths_hosp_", deaths_ages)
+  nms_deaths_aggr <- c("deaths", "deaths_hosp")
```

Comment:
I think you can get rid of this - was removed in my PR

Reply:
ok, done